<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ihm_DataView</class>
 <widget class="QMainWindow" name="ihm_DataView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="active_trace">
      <property name="text">
       <string>On/Off</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="pb_up">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>Monte la variable sélectionnée dans la liste</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/icons/up.png</normaloff>:/icons/up.png</iconset>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="pb_down">
      <property name="toolTip">
       <string>Descend  la variable sélectionnée dans la liste</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/icons/down.png</normaloff>:/icons/down.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="pb_enregistrer_trace">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>Enregistrer la trace dans un fichier</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QPushButton" name="pb_effacer_liste">
      <property name="toolTip">
       <string>Effacer le contenu de la trace</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/icons/stop.png</normaloff>:/icons/stop.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QListWidget" name="lisye_variables">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <property name="uniformItemSizes">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Nouvel élément</string>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="4" colspan="4">
     <widget class="QTableWidget" name="table_variables_valeurs">
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>115</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>120</number>
      </attribute>
      <column>
       <property name="text">
        <string>Nom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Valeur</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QLineEdit" name="valueToWrite">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Appuyer sur &quot;Entrée&quot; pour valider la saisie</string>
      </property>
      <property name="statusTip">
       <string>Modifie la ou les variables sélectionnées dans la liste des variables observées</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QLabel" name="valueToWrite_label">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Write</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="choix_mode_inspection">
      <item>
       <property name="text">
        <string>Instantané</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Temporel</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="PB_refresh_liste">
      <property name="toolTip">
       <string>Mise à jour de la liste des variables disponibles</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
